site/src/utils/markdown-styles.js

```diff
@@ -90,32 +96,29 @@ export const h2 = styled.h2`
   }
 `
 
-// $FlowFixMe(flow@0.100.0): tagged templates don't support generics
 export const h3 = styled.h3`
   font-size: 20px;
   padding-top: 20px;
   ${baseHeadingStyles};
 `
 
-// $FlowFixMe(flow@0.100.0): tagged templates don't support generics
 export const h4 = styled.h4`
   font-size: 16px;
   margin-top: 22px;
   padding-top: 15px;
   ${baseHeadingStyles};
 `
 
-// $FlowFixMe(flow@0.100.0): tagged templates don't support generics
 export const h5 = styled.h5`
   font-size: 14px;
 `
 
-// $FlowFixMe(flow@0.100.0): tagged templates don't support generics
 export const h6 = styled.div`
   font-size: 12;
 `
 
-export const a = styled.a<empty>(
+export const a = styled.a(
+  /* <empty> */
```

Comment:
according to the PR description all `<empty>` should be removed

Reply:
Ha, I missed some... will do another pass.